src/components/choice/_choices.scss

```diff
@@ -0,0 +1,63 @@
+.choice {
+
+    &-ans {
+        width: 80;
+        display: block;
+        text-align: center;
+        margin-left: -1rem;
+        margin-top: 0.2rem;
+        margin-bottom: 0.3rem;
+
+        &-linked {
+            width: 8%;
+            //margin-left: -2rem;
+            background-color: whitesmoke;    
+            font-family: 'Roboto Mono Medium';
+            font-size: large;
+            padding: 0.3rem;
+            height: 1.5rem;
+            //border-radius: 0.5rem;
+            border: solid black 1px;
+            margin-right: 1rem;
+            //padding: 0.5rem;
+            vertical-align: middle;
+
+            &:focus{
+                outline: none;
+                background-origin: padding-box;
+                border: solid silver 0.1rem;
+                box-shadow: 0.5px 0.5px; 
+            }
+        }
+
+        &-rated {
+            height: 2rem;
+        }
+    }
+
+    &-content {
+        resize: none;
+        width: 60%;
+        font-family: 'Times New Roman';
```

Comment:
font-family property should be same of all the project. It may reason that weird UI.